test/blueprints/Do.Test.Blueprints.Service/Business/Singleton.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Do.Communication;
+using Do.Database;
+using Newtonsoft.Json;
+
+namespace Do.Test;
+
+public class Singleton(
+    TimeProvider _timeProvider,
+    Func<OperationWithGenericParameter<Entity>> _newOperationWithGenericParameter
+) : SingletonBase, IInterface
+{
+    public override DateTime GetTime() =>
+        _timeProvider.GetNow();
```

Comment:
this is confusing, it also exists in authentication, to test DI you may keep this here, but rename the one in authentication

Reply:
but still, this `GetTime` is not a test about `Singleton`, move it to a class named `InjectDependency` or something, test it there

Reply:
actually injecting `TimeProvider` for dependency injection test is a bit confusing, you better create a `Service` and another `InjectedService` class, inject one to another to test it. it will be simpler